compiler/rustc_transmute/src/layout/tree.rs

```diff
@@ -268,6 +384,10 @@ pub(crate) mod rustc {
                     }))
                 }
 
+                ty::Char => Ok(Self::char(cx.tcx().data_layout.endian.into())),
+
+                ty::Alias(_kind, ty) => Self::from_ty(ty.to_ty(cx.tcx()), cx),
```

Comment:
```suggestion
                ty::Alias(_kind, ty) => Self::from_ty(/* normalize ty */),
```
@tmiasko / @compiler-errors, what's the current way to normalize a type when we *do* care about regions?

Reply:
If we're punting on normalization, can you remove this branch so we don't regress projections to an infinite loop?

Reply:
> If we're punting on normalization, can you remove this branch so we don't regress projections to an infinite loop?

Done.

> @tmiasko / @compiler-errors, what's the current way to normalize a type when we _do_ care about regions?

Maybe, for the time being (but in a separate PR), we could at least support `'static` types by normalizing without caring about regions and returning an error when the type is non-`'static`? That would at least cover the `NonZero` case.